README.md

```diff
@@ -60,6 +60,7 @@ DRIVING_FERRY
 PUBLIC_TRANSPORT
 WALKING_FERRY
 CYCLING_FERRY
+DRIVING
```

Comment:
Should this also include WALKING and CYCLING?

Reply:
Yes it should, on it

Reply:
Yes it should, on it